fabfile.py

```diff
@@ -14,18 +14,12 @@
 
 DEPLOY_ROOT = "/opt"
 
-# Node 18 doesn't seem to work on RHEL 7.
-# https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V18.md#toolchain-and-compiler-upgrades
-NODE_VERSION = "16"
+NODE_VERSION = "18"
```

Comment:
Can we use Node 20 or is that crazy talk?

Reply:
Not crazy at all, I will try it.